lib/modules/manager/helmv3/utils.ts

```diff
@@ -82,8 +82,19 @@ export function isAlias(repository: string): boolean {
   return repository.startsWith('@') || repository.startsWith('alias:');
 }
 
-export function isOCIRegistry(repository: Repository): boolean {
-  return repository.repository.startsWith('oci://');
+export function isOCIRegistry(
+  repository: Repository | string | undefined
+): boolean {
+  if (typeof repository === 'undefined') {
+    return false;
+  }
+
+  let string = repository;
+  if (typeof repository === 'object') {
+    string = repository.repository;
+  }
+
+  return (string as string).startsWith('oci://');
```

Comment:
```suggestion
export function isOCIRegistry(
  repository: Repository | string | null | undefined
): boolean {
  if (is.nullOrUndefined(repository)) {
    return false;
  }
  const repo = is.string(repository) ? repository : repository.repository;
  return repo.startsWith('oci://');
```
You can use the `is` library for typesafety. 
you still have to add the import as I'm not able to directly edit your files. 
`import is from '@sindresorhus/is';`